DigitalLearningSolutions.Web/Helpers/NewlineSeparatedStringListHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    public static class NewlineSeparatedStringListHelper
+    {
+        public static (string, List<string>) RemoveStringFromNewlineSeparatedList(string list, int index)
+        {
+            var options = SplitNewlineSeparatedList(list);
+            options.RemoveAt(index);
+            return (JoinNewlineSeparatedList(options), options);
+        }
+
+        public static (string, List<string>) AddStringToNewlineSeparatedList(string? list, string newItem)
+        {
+            var options = list != null ? SplitNewlineSeparatedList(list) : new List<string>();
+            options.Add(newItem);
+            return (JoinNewlineSeparatedList(options), options);
+        }
+
+        public static List<string> SplitNewlineSeparatedList(string? list)
+        {
+            return list == null ? new List<string>() : list.Split("\r\n").ToList();

```

Comment:
Super minor, but should we pull the separator string into a const at the top of the file so if we ever want to change the separator, it only needs to be done in one place?